ethpm_types/source.py

```diff
@@ -13,41 +15,72 @@ class Compiler(BaseModel):
 
 
 class Checksum(BaseModel):
-    algorithm: str
-    hash: str
+    """Checksum information about the contents of a source file."""
+
+    algorithm: Algorithm
+    hash: Hex
 
 
 class Source(BaseModel):
+    """Information about a source file included in a Package Manifest."""
+
+    """Array of urls that resolve to the same source file."""
```

Comment:
Are these docs looking okay when built?

I think they actually have to be underneath the property for some weird reason. I have tried to put them above before and concluded that it didn't work.

Reply:
7297d6b